upgrade/sql/1.7.0.0.sql

```diff
@@ -8,6 +8,8 @@ INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VAL
   ('PS_ORDER_RECALCULATE_SHIPPING', '1', NOW(), NOW()),
   ('PS_MAINTENANCE_TEXT', 'We are currently updating our shop and will be back really soon.&lt;br&gt;Thanks for your patience.', NOW(), NOW());
 
+/* PHP:ps_1700_short_description_config(); */;
```

Comment:
Why use PHP when you can use `INSERT IGNORE INTO PREFIX_configuration` ?

Reply:
@PierreRambaud 

You sure? I want to do it only if it's necessary

Reply:
I think I misread something.
You're trying to update only because the 1.6 had a value of 0?
The check is not obvious imho, I was thinking you tried to reach the parameter, if it doesn't exists you create it :sweat_smile:

Reply:
@PierreRambaud, some 1.6 installations had  "0" as a value, and we need to update this after the upgrade. If not, people can't save the description of the product 👍

Reply:
Maybe it will be more obvious if we do
```
return (0 === Configuration::get('PS_PRODUCT_SHORT_DESC_LIMIT'))
? Configuration::updateValue('PS_PRODUCT_SHORT_DESC_LIMIT', 800) : true;
```

Reply:
Agree with @matks

Reply:
@PierreRambaud @matks I've just changed this condition for you guys ❤️ 

btw. PHPStan is not happy with 1.6, I think he doesn't like such old code ;-P